packages/selector-ui/src/components.d.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable */
```

Comment:
I'm wondering if there's a way to consolidate these autogenerated files we can ignore them in git. Ideally these files shouldn't be under source control